cns/middlewares/k8sSwiftV2_linux.go

```diff
@@ -30,50 +32,12 @@ func (k *K8sSWIFTv2Middleware) setRoutes(podIPInfo *cns.PodIpInfo) error {
 		routes = append(routes, virtualGWRoute, route)
 
 	case cns.InfraNIC:
-		// Get and parse infraVNETCIDRs from env
-		infraVNETCIDRs, err := configuration.InfraVNETCIDRs()
+		// get service and infravnet routes
```

Comment:
Could you modify this comment-- https://github.com/Azure/azure-container-networking/compare/jainriya/npmliteCNSchanges...coderefactorcnsswiftv2#diff-70deeff65e796118967cdd314e7812a08e0fba22d64f116eaf098a50a250642bR35 and this one https://github.com/Azure/azure-container-networking/compare/jainriya/npmliteCNSchanges...coderefactorcnsswiftv2#diff-bc799fe974a6b7f59d815d1e4123ea023fec16051cba9edb16815e8d36973823R39 to be more explicit with what they are doing since the current comment makes it seem like they are doing the same thing, but in reality the linux one is doing more (adds in pod cidrs too)